scylla/src/transport/iterator.rs

```diff
@@ -69,17 +70,6 @@ struct ReceivedPage {
     pub tracing_id: Option<Uuid>,
 }
 
-pub(crate) struct PreparedIteratorConfig {
```

Comment:
Why did you decide to delete the whole structure? If you wanted to remove some fields from `PreparedIteratorConfig` then you should have done just that. Removing the whole `PreparedIteratorConfig` introduces a lot of noise because you have to change all the lines that used to refer to the `PreparedIteratorConfig`'s fields.

If we wanted to remove `PreparedIteratorConfig` then it should have been done in a separate commit. In my opinion this structure is fine as otherwise the `new_for_prepared_statement` function would still have to take a large number of arguments.